### PR TITLE
[change] Removed squashed migrations #373

### LINKED ISSUE #373: [change] Remove squashed migrations
The app `openwisp_monitoring.monitoring` has a number of squashed migrations, we should follow the recommendations given by Django about how to remove squashed migrations and do so.

### PR DESCRIPTION
**Pull request by Aryamanz29**
- Removed squashed migrations of `openwisp_monitoring.monitoring`

Closes #373

Acc to `Django docs` (transition the squashed migration to a normal migration)

- [x]  Deleting all the migration files it replaces.

- [x] Updating all migrations that depend on the deleted migrations to depend on the squashed migration instead.

- [x] Removing the `replaces` attribute in the Migration class of the squashed migration (this is how Django tells that it is a squashed migration).

Checks:

- [x] I have manually tested the proposed changes
- [ ] I have written new test cases to avoid regressions (if necessary)
- [ ] I have updated the documentation (e.g. README.rst)

### CODE REVIEW

**Comment by Aryamanz29** [comment]
WIP

**Comment by Aryamanz29** [comment]
There are some more squashed migrations in `device` and `check`, do we need to remove them as well ?
- https://github.com/openwisp/openwisp-monitoring/tree/master/openwisp_monitoring/device/migrations
- https://github.com/openwisp/openwisp-monitoring/tree/master/openwisp_monitoring/check/migrations

**Review by nemesisdesign — COMMENTED** [review]
> There are some more squashed migrations in `device` and `check`, do we need to remove them as well ?
> 
> * https://github.com/openwisp/openwisp-monitoring/tree/master/openwisp_monitoring/device/migrations
> * https://github.com/openwisp/openwisp-monitoring/tree/master/openwisp_monitoring/check/migrations

Yes please, let's remove all squashed migrations.

File: openwisp_monitoring/check/migrations/0003_rename_active_to_is_active.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 class Migration(migrations.Migration):
 
-    dependencies = [('check', '0003_create_ping')]
+    dependencies = [('check', '0002_create_ping')]
```
[the remark this answers is not in the document]
Comment (Aryamanz29) [reply openwisp_monitoring/check/migrations/0003_rename_active_to_is_active.py:10]: My bad

[the remark this answers is not in the document]
Comment (nemesisdesign) [reply openwisp_monitoring/check/migrations/0003_rename_active_to_is_active.py]: Ok, thanks for reporting, I created https://github.com/openwisp/openwisp-controller/pull/655

**Review by nemesisdesign — APPROVED** [review]
Thanks :+1:

**Pull request merged**